sdk/appconfiguration/app-configuration/src/policies/throttlingRetryPolicy.ts

```diff
@@ -0,0 +1,129 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import {
+  BaseRequestPolicy,
+  RequestPolicy,
+  RequestPolicyOptions,
+  RequestPolicyFactory,
+  WebResource,
+  HttpOperationResponse,
+  Constants,
+  delay,
+  RestError
+} from "@azure/core-http";
+
+export function throttlingRetryPolicy(): RequestPolicyFactory {
```

Comment:
General: Add @ignore to anything that is exported but shouldn't be documented (for TypeDoc's sake).

We're not exposing the policies to users right?

Reply:
These should be considered internal for now.